### PR TITLE
Public workflows

### PR DESCRIPTION
**Pull request by dvargas92495**
Steps left to productionize:
- [x] Setup all examples in common architectures
- [x] Fix theming
- [x] Make minimap smaller
- [x] Resolve 401 errors
- [x] Address loading on mobile
- [x] Full screen views of many nodes are squished
- ~[ ] IFrame full screen button~ deferring

https://app.shortcut.com/vellum/story/2956

### CODE REVIEW

File: fern/docs/content/help/workflows/examples.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Workflow Examples
+description: Check out some of our live embedded public workflow examples below!
+---
+
+## Function Calling Demo
+
+<iframe
+  src="https://samepage.ngrok.io/share/workflow-sandboxes/c7bf0a38-19cf-4e31-8bfb-a237d8daba67"
```
Comment (dvargas92495) [inline fern/docs/content/help/workflows/examples.mdx:9]: will point to `https://app.vellum.ai` when ready, had this pointing to localhost for demo-ing purposes

File: fern/docs/content/help/workflows/common-architectures.mdx [hunk 2]
```diff
@@ -28,39 +32,75 @@ If you’re building an agent that answers questions coming from users (e.g., a
...
+- Step 4: We use a Conditional Node to dictate when to exit the loop.
+- Step 5: We feed the `"item"` key from the `iterator` into the Prompt Node we care about executing on each iteration.
+- Step 6: We use a Code Execution node to append an output on each iteration.
+- Step 7: Finally, the Templating Node will return the output data returned by the previous step's Code Execution node.
 
 ## 4. Retrying a Prompt Node in case of non-deterministic failure
 
 Prompt nodes support two selectable outputs - one from the model in case of a valid output and one in case of a non deterministic error. Model hosts fail for all sorts of reasons that include timeouts, rate limits, or server overload. You could make your production-grade LLM features resilient to these features by adding retry logic into your Workflows!
 
-![Workflow Retry Example](https://storage.googleapis.com/vellum-public/help-docs/workflow_retry_example.png)
```
Comment (dvargas92495) [inline fern/docs/content/help/workflows/common-architectures.mdx:63]: https://www.loom.com/share/6398e7494c6a4e2fadc88f1ae21e33b5

**Comment by dvargas92495** [comment]
Ready for review

**Review by noanflaherty — APPROVED** [review]
Agree that we can punt on full-screen mode and clone to workspace. This is all so dope though

**Pull request merged**